tests/pfc/files/configs/pfc.py

```diff
@@ -0,0 +1,399 @@
+                  "Number of physical ports must be at least 2")
+
+    configs = []
+    for i in range(len(available_phy_port)):
+        rx_id = i
+        tx_id = (i + 1) % len(available_phy_port)
+
+        phy_tx_port = get_location(available_phy_port[tx_id])
+        phy_rx_port = get_location(available_phy_port[rx_id])
+
+        #########################################################################
+        # common L1 configuration
+        #########################################################################
+
+        tx = Port(name='Tx', location=phy_tx_port)
+        rx = Port(name='Rx', location=phy_rx_port)
+
+        pfc = Ieee8021qbb(pfc_delay=1,
+                          pfc_class_0=0,
+                          pfc_class_1=1,
+                          pfc_class_2=2,
+                          pfc_class_3=3,
+                          pfc_class_4=4,
+                          pfc_class_5=5,
+                          pfc_class_6=6,
+                          pfc_class_7=7)
+
+        flow_ctl = FlowControl(choice=pfc)
+
+        l1_oneHundredGbe = OneHundredGbe(link_training=True,
+                                         ieee_media_defaults=False,
+                                         auto_negotiate=False,
+                                         speed='one_hundred_gbps',
+                                         flow_control=flow_ctl,
+                                         rs_fec=True)
+
+        common_l1_config = Layer1(name='common L1 config',
+                                  choice=l1_oneHundredGbe,
+                                  port_names=[tx.name, rx.name])
+
+        config = Config(ports=[tx, rx],
+            layer1=[common_l1_config],
+            options=Options(PortOptions(location_preemption=True)))
+
+        configs.append(config)
+
+    return configs
+
+
+@pytest.fixture
+def lossy_configs(conn_graph_facts,
+                  duthost,
+                  lossless_prio_dscp_map,
+                  one_hundred_gbe,
+                  start_delay,
+                  traffic_duration,
+                  pause_line_rate,
+                  traffic_line_rate,
+                  frame_size, 
+                  serializer) :
+
+    for p in lossless_iteration_list(lossless_prio_dscp_map) :
+        yield (base_configs(conn_graph_facts=conn_graph_facts,
+                            duthost=duthost,
+                            lossless_prio_dscp_map = p,
+                            one_hundred_gbe=one_hundred_gbe,
+                            traffic_duration=traffic_duration,
+                            start_delay=start_delay,
+                            pause_line_rate=pause_line_rate,
+                            traffic_line_rate=traffic_line_rate,
+                            pause_frame_type='priority',
+                            frame_size=frame_size,
+                            serializer=serializer))
+
+
+@pytest.fixture
+def global_pause(conn_graph_facts,
```

Comment:
Why we need this?

Reply:
This file is shared by two test cases 1) test_pfc_pause_lossy.py 2) test_global_pause.py. So fixture used by both the test cases exists here. This particular function is used by test_global_pause.py. So this should be here.